src/transformers/models/bart/modeling_tf_bart.py

```diff
@@ -668,60 +671,79 @@ def call(
             return_dict (:obj:`bool`, `optional`):
                 Whether or not to return a :class:`~transformers.file_utils.ModelOutput` instead of a plain tuple.
         """
-        if input_ids is not None and inputs_embeds is not None:
+        inputs = input_processing(
```

Comment:
made `BartDecoder` and `BartEncoder` work similar to how it's done in TFT5 - lemme know if this all looks like you'd expect @jplu